docs/hopr-documentation/versioned_docs/version-v1.86/staking/how-to-stake.md

```diff
@@ -3,27 +3,27 @@ id: how-to-stake
 title: Staking on HOPR smart contract
 ---
 
-HOPR staking scheme goal is to reward HOPR token holders while providing a way to develop and test cover traffic with the help of the community. This is a six month-scheme takes standard yield farming and gives a gamified twist thanks to collectible NFTs which can boost your APY. It was launched on July 27th and will run until 17th January 2022, a period of 175 days.
+HOPR staking scheme goal is to reward HOPR token holders while providing a way to develop and test cover traffic with the help of the community. This staking scheme takes standard yield farming and gives a gamified twist thanks to collectible NFTs which can boost your APR. First Staking Season launched on July 27th 2021, at the moment we are running Staking Season 3, which started on April 26th and will run until June 26th, a period of 90 days.
```

Comment:
"The first staking season launched on July 27th, currently we're running staking season 3, which started on April 26th 2022 and will run for a period of 90 days."